CHANGELOG.md

```diff
@@ -33,6 +33,10 @@ this project adheres to
   multiple removals.
 - Updated review-database to 0.37.0, which fixes event filtering by multiple IP
   addresses for `ExternalDdos`, `MultiHostPortScan`, and `RdpBruteForce`.
+- Changed behavior of blocklist, allowlist and trusted user agents updates to
+  automatically broadcast without manual apply. As a result of this change, the
+  GraphQL APIs that handle manual apply have been removed.
+  (`applyBlockNetworks`, `applyAllowNetworks`, `applyTrustedUserAgent`)
```

Comment:
Even though the removed APIs are mentioned, the changed ones are not. This could be inconvenient for users, as they would have to manually figure out which APIs have been modified.

Reply:
I have added about the modified APIs to the CHANGELOG.